content/english/Kusto-KC7/activity-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Activity 0: Login & Setup of the Cyber Environment"
+description: ""
+draft: false
+weight: 4
+---
+
+{{< youtube VL7fJr54CG0 >}}
+
+# Getting Started
+
+Okay Cyber Defenders! The first thing you need to do is set up the environment in which you will be hunting hackers. In order to do that, open the links below and follow the directions on first setting up Azure Data Explorer (ADX) and then logging into the Scoreboard session.
+- Azure Data Explorer : https://dataexplorer.azure.com/ 
+- The Scoreboard : https://aka.ms/kc7scoreboard 
+- (Optional) PDF Guide of This Workshop : https://kc7cyber.com/guide1 
+
+
+
+<!-- ![resource](../kusto-kc7/Images/Resources.png) -->
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/Resources.png?raw=true" alt= “” width="80%" height="value">
```

Comment:
Please consider providing alt text for all the images to meet accessibility guidelines.

```suggestion
<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/Resources.png?raw=true" alt="Links to the three resources that will be used during the workshop" width="80%" height="value">
```

Reply:
UPdated